JavaScript/Topics/20.GraphQL/1.Basics/README.md

```diff
@@ -0,0 +1,198 @@
+# GraphQL - Graph Query Language
+
+## History of GraphQL
+`GraphQL` was released in 2015 by Facebook. It has offered a new way and a promising alternative to the traditional REST API. Many companies including Github, Shopify, Intuit and many more have adopted GraphQL into their ecosystems. It's demand is increasing drastically and projected to grow exponentially over the new decade. 
+
+**Note:- We will use an array as a database and will build API using [Node.js](https://nodejs.org/en/) with [Appolo Server](https://www.apollographql.com/docs/apollo-server/)**
```

Comment:
Please correct **Array is not a database it's a data type** @its-me-Harsh-Anand :-)

Reply:
Okay mam, I'll change, but I was using Array as a database to store data instead of using mongoDB or firebase